README.md

```diff
@@ -5,9 +5,10 @@
 _(Note: Ayo.js is forked from [Node.js][]. Currently, a lot of the documentation
 still points towards the Node.js repository.)_
 
-Ayo.js is a JavaScript runtime built on Chrome's V8 JavaScript engine. Ayo.js
+Ayo.js is a JavaScript runtime built on Chrome's V8 JavaScript engine. It
 uses an event-driven, non-blocking I/O model that makes it lightweight and
-efficient. The Ayo.js package ecosystem, [npm][], is the largest ecosystem of
+efficient. Ayo.js shares with the rest of the JavaScript
+implementations the [`npm` package ecosystem][], the largest set of
```

Comment:
reads a tad strange to me, but I'm not sure how to word it better off-hand

Reply:
perhaps `Ayo.js, like the rest of the JavaScript implementations, benefits from the npm package ecosystem`?